### ISSUE TITLE
Support '-e' / '--execute' to run queries

### ISSUE THREAD

**Issue by kenden** [dbcli/mycli#286]
The mysql client allows to run queries and return the results with the --execute param:
```
$ mysql --host 127.0.0.1 --user root -proot --database mysql --execute='show tables;'
mysql: [Warning] Using a password on the command line interface can be insecure.
+---------------------------+
| Tables_in_mysql           |
+---------------------------+
| columns_priv              |
| db                        |
| event                     |
| func                      |
| general_log               |
| help_category             |
| help_keyword              |
| help_relation             |
| help_topic                |
| innodb_index_stats        |
| innodb_table_stats        |
| ndb_binlog_index          |
| plugin                    |
| proc                      |
| procs_priv                |
| proxies_priv              |
| servers                   |
| slave_master_info         |
| slave_relay_log_info      |
| slave_worker_info         |
| slow_log                  |
| tables_priv               |
| time_zone                 |
| time_zone_leap_second     |
| time_zone_name            |
| time_zone_transition      |
| time_zone_transition_type |
| user                      |
+---------------------------+

```

It would be nice to get the same for mycli.

Workaround for now:

```
$ echo 'show tables;' | mycli --host 127.0.0.1 --user root -proot --database mysql
Tables_in_mysql
columns_priv
db
event
func
general_log
help_category
help_keyword
help_relation
help_topic
innodb_index_stats
innodb_table_stats
ndb_binlog_index
plugin
proc
procs_priv
proxies_priv
servers
slave_master_info
slave_relay_log_info
slave_worker_info
slow_log
tables_priv
time_zone
time_zone_leap_second
time_zone_name
time_zone_transition
time_zone_transition_type
user

**Comment by mdsrosa** [comment]
@kenden That seems a fair request. I'll look into it. Thanks! 😉

**Comment by mdsrosa** [comment]
@kenden Which version of `mycli` are you using? 
Since version `1.7.0` you can run queries in stdin batch mode like this
`mycli < test.sql` or `mysql --host 127.0.0.1 --user root -proot --database mysql < test.sql`. 

Would that satisfy your needs for the moment?

**Comment by amjith** [comment]
This is now available in version 1.8.0. Please upgrade mycli: 

```
pip install -U mycli
```
